onfido-java/src/main/java/com/onfido/webhooks/WebhookEvent.java

```diff
@@ -15,17 +16,21 @@ public class WebhookEvent {
   @Json(name = "object")
   private final WebhookEventObject object;
 
+  @Json(name = "resource")
+  private final Task resource;
```

Comment:
The `resource` object can't be a `Task` otherwise it will break the other webhooks events that also pass the `resource` in the webhook payload, for example the `workflow_run_completed` event.

My suggestion would be to add a generic `Map` in here instead of using `Task`

Reply:
We also have the [watchlist_monitor.matches_updated](https://documentation.onfido.com/#watchlist-monitor-matches-updated-webhook-event-object) event where the resource object is not as similar.

So yes, please change it to a generic Object instead